content/api-specs/madagascar/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Madagascar Release API Specification
+menuTitle: Madagascar (v1.7.0)
+description: "API Specification for the Madagascar release (v1.7.0) of various SODA Projects"
+weight: 5
```

Comment:
@joseph-v This should be 3. the lighter weight bubbles up in the menu. We put the latest release at the top. The Lamu release API spec has weight 4.

Reply:
Done